packages/libs/lambda-at-edge/rollup.config.js

```diff
@@ -21,6 +22,10 @@ const generateConfig = (input) => ({
     format: "cjs"
   },
   plugins: [
+    del({
+      targets:
+        './dist/${input.filename}/${input.minify ? "minified" : "standard"}'
```

Comment:
' should be `?

Reply:
Indeed, I managed to break it after testing 🤦🏽 fixed